code/game/turfs/simulated/floor/light_floor.dm

```diff
@@ -103,6 +102,16 @@
 	else
 		to_chat(user, "<span class='warning'>[src]'s light bulb appears to have burned out.</span>")
 
+/turf/simulated/floor/light/proc/toggle_light(light)
+	// 0 = OFF
```

Comment:
I'd say that this is logical due to how the proc is set up. Not sure if the comments add something